src/TConsole.cpp

```diff
@@ -673,19 +679,23 @@ void TConsole::resizeEvent(QResizeEvent* event)
     int x = event->size().width();
     int y = event->size().height();
 
+    int scrollBarHeight = 0;
+    if (mpHScrollBar && mpHScrollBar->isVisible()) {
+        scrollBarHeight = 15;
```

Comment:
This might not be safe - a scrollbar could be styled by the OS to be a different size. Is there a way to get the real value?

Reply:
We set the same size for the vertical scrollbar BTW - and if you do not specify a size it can grow to fill the layout it is in - when I was playing around and trying to clean up the *messy* layout done manually in the `TConsole` constructor (to restructure things to):
```
Grid Layout:
    |                 |
Col:|       0         | 1
    |                 |
Row:|                 |
--- +--------------------+
    |+---------------++-+|
    ||               ||V||
    ||               || ||
    ||    Upper      ||S||
 0  ||     Pane      ||-||
    ||               ||B||
    ||               ||a||
    ||               ||r||
    |+---------------++-+|
--- |+---------------+XXX|
 1  ||   H S-bar     |XXX|
    |+---------------+XXX|
--- |+------------------+|
    ||                  ||
    ||    Lower         ||
 2  ||     Pane         ||
    ||                  ||
    |+------------------+|
    +--------------------+
===================

Note: the command line stuff (and buttons and Latency/Time) are in a separate layout in a parent of this widget
```

In the absence of more explicit settings column 1 and row 1 will adopt the same width as column 0 and row 0 (and 2 but that is regulated)...